src/content/en/fundamentals/design-and-ui/input/touch/_code/states-example.html

```diff
@@ -1,33 +1,21 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
```

Comment:
These are still missing across a number of files, can you take a peek again please.